src/social/service/comment.service.ts

```diff
@@ -56,10 +60,30 @@ export class CommentService {
     return comment;
   }
 
-  async getCommentsByUserId(userId: string): Promise<Comment[]> {
+  async validateNumber(limitOffset, warning: string): Promise<void> {
```

Comment:
See if you can use ValidationPipe in controller to validate instead
```javascript
import {ValidationPipe} from '@nestjs/common'
@Get()
@UsePipes(ValidationPipe)
async get(@Query() query: GetCatQuery) {
  //....
}

class GetCatQuery {
    @IsNumber()
    @Min(0)
    @Transform(value => Number(value))
    offset: number;
}
```
https://github.com/nestjs/nest/issues/766

Reply:
Rebased